src/Project.hs

```diff
@@ -175,6 +175,8 @@ data ApprovedFor
 data MergeCommand
   = -- | The PR should be approved for merging and optionally deploying.
     Approve ApprovedFor
+  | -- | The PR is approved for merging and was marked as a hotfix
```

Comment:
If this is mutually exclusive with approving on friday, wouldn't it make more sense to treat this as a `MergeWindow` just like merging on Friday instead of duplicating `Approve`? It is a merge window, and then you can also support `retry hotfix` properly.

Reply:
Thought about this a little, and I agree. I changed the syntax to `<command> as hotfix` now